compiler-rt/test/ubsan_minimal/TestCases/override-callback.c

```diff
@@ -0,0 +1,14 @@
+// RUN: %clang -fsanitize=implicit-integer-sign-change %s -o %t && %run %t 2>&1 | FileCheck %s --check-prefixes=CHECK
+
+#include <stdio.h>
+#include <stdint.h>
+
+void __ubsan_handle_implicit_conversion_minimal() {
```

Comment:
I think it's going to be hard to maintain on client size, when we add more types of checks

I believe it should be single function with simple interface.

Maybe just:
`void SANITIZER_CDECL
__sanitizer_report_error_summary(const char *error_summary);`

With exactly the same format of output.
"SUMMARY: ubsan: morestuff"

Reply:
I renamed report_this_error to __sanitizer_report_ubsan_error so we have a single handler now. PTAL.

Reply:
There is function declared as:

```
void SANITIZER_CDECL
__sanitizer_report_error_summary(const char *error_summary);
```

if we abandon callerpc we can use that one
Do you see good usecase of it?

Reply:
```
decorate_msg(MSG_TMPL_END(msg_buf, msg), caller);                          \
    if (report_this_error(caller)) {                                           \
      message(msg_buf);                                                        \
    }                                                                          \
    __sanitizer_report_error_summary(msg_buf);                                 \
```

Reply:
Actually just 

```
NTERFACE void __ubsan_handle_##name##_minimal() {                           \
    uintptr_t caller = GET_CALLER_PC();                                        \
    if (!report_this_error(caller))                                            \
      return;                                                                  \
    char msg_buf[MSG_BUF_LEN(msg)] = MSG_TMPL(msg);                            \
    decorate_msg(MSG_TMPL_END(msg_buf, msg), caller);                          \
    __sanitizer_report_error_summary(msg_buf);                                 \
  }
```


With default implementation of     __sanitizer_report_error_summary as write